src/components/NewGalaxyModal.js

```diff
@@ -37,394 +37,397 @@ const maxNodepoolSize = 1000 // per zone according to https://cloud.google.com/k
               }, ['Delete']),
-              h(ButtonSecondary,
-                { disabled: true, style: { marginRight: '1rem' }, tooltip: 'Cloud Compute must be running.', onClick: pauseGalaxy },
-                ['Pause'])
-            ])
-          }]
-        )]
-    )
-  }
+              resumeButton
+            ])],
+            ['ERROR', () => deleteButton],
+            [Utils.DEFAULT, () => {
+              return h(Fragment, { tooltip: 'Cloud Compute must be resumed first.' }, [
+                h(ButtonSecondary, {
+                  disabled: true, style: { marginRight: 'auto' }, tooltip: 'Cloud Compute must be running.', onClick: () => setViewMode('deleteWarn')
+                }, ['Delete']),
+                h(ButtonSecondary,
+                  { disabled: true, style: { marginRight: '1rem' }, tooltip: 'Cloud Compute must be running.', onClick: pauseGalaxy },
+                  ['Pause'])
+              ])
+            }]
+          )]
+      )
+    }
 
-  const renderMessaging = () => {
-    return Utils.switchCase(viewMode,
-      ['createWarn', renderCreateWarning],
-      ['deleteWarn', renderDeleteDiskChoices],
-      ['launchWarn', renderLaunchWarning],
-      [Utils.DEFAULT, renderDefaultCase]
-    )
-  }
+    const renderMessaging = () => {
+      return Utils.switchCase(viewMode,
+        ['createWarn', renderCreateWarning],
+        ['deleteWarn', renderDeleteDiskChoices],
+        ['launchWarn', renderLaunchWarning],
+        [Utils.DEFAULT, renderDefaultCase]
+      )
+    }
 
-  const renderCreateWarning = () => {
-    return div({ style: styles.drawerContent }, [
-      h(TitleBar, {
-        title: 'Cloud Environment',
-        style: { marginBottom: '0.5rem' },
-        onDismiss,
-        onPrevious: !!viewMode ? () => setViewMode(undefined) : undefined
-      }),
-      div({ style: { marginBottom: '1rem' } }, ['Cloud environments consist of application configuration, cloud compute and persistent disk(s).']),
-      div({ style: { ...styles.whiteBoxContainer, backgroundColor: colors.accent(0.1), boxShadow: Style.standardShadow } }, [
-        div({ style: { flex: '1', lineHeight: '1.5rem', minWidth: 0, display: 'flex' } }, [
-          span({ style: { marginRight: '0.5rem', marginTop: '0.5rem' } }, [icon('clockSolid', { size: 25, color: colors.accent() })]),
-          div([
-            div({ style: { ...styles.headerText, marginTop: '0.5rem' } }, ['Set up duration']),
-            div({ style: { lineHeight: 1.5 } }, [
-              div(['Creating a cloud environment for Galaxy takes ', span({ style: { fontWeight: 600 } }, ['8-10 minutes.'])]),
-              div(['You can navigate away, and we will notify you when it\'s ready. '])
+    const renderCreateWarning = () => {
+      return div({ style: styles.drawerContent }, [
+        h(TitleBar, {
+          title: 'Cloud Environment',
+          style: { marginBottom: '0.5rem' },
+          hideCloseButton: isAnalysisMode,
```

Comment:
we hide the inner navigation pane's close in all cases in this modal when its nested in the create analysis flow